openedx/core/djangoapps/catalog/utils.py

```diff
@@ -66,6 +72,19 @@ def get_programs(uuid=None, type=None):  # pylint: disable=redefined-builtin
         return []
 
 
+def get_program_type(name):
```

Comment:
That's fair. At the very least, you don't need to add a new function for this. You can extend the existing `get_program_types` function so that it only returns the named type object.